pages/rdm_cycle/planning.md

```diff
@@ -22,4 +45,26 @@ Data management planning will facilitate the progress of your project, as you ca
 
 ## Related topics
 
+* [Collect](collecting)
+* [Processing](processing)
+* [Analyse](analysing)
+* [Preserving](preserving)
+* [Sharing](sharing)
+* [Find and reuse existing data](reusing)
+* File format
+* Data volume
+* Licences
+* [Metadata Management and data documentation](metadata_management)
+* Ontology
+* Data organisation
+* Identifiers
+* Data quality
+* [Storage](storage)
+* Data repositories
+* Costs for data management, storage and sharing
+* Responsibilities
+* Ethical and legal issues
+* [Security and privacy](security_and_privacy)
+* Funder quidelines
```

Comment:
It looks like almost all the pages are listed. Does it still fulfill a purpose than?

Reply:
We thought about this with korbinian, it is tricky.. Planning is a stage to which everything is connected. If something is left out, the question remains where would we draw the line? I don't know what is the best thing to do!

Reply:
I would suggest to keep all the related topics. - Without a DMP tool this would reflect a kind of checklist. Maybe this is one of the reasons  why we recommend a guiding DMP tool (e.g. DSW)